src/components/pages/Landing/Landing.js

```diff
@@ -1,16 +1,25 @@
 import React from 'react';
 import 'antd/dist/antd.css';
+import { Link } from 'react-router-dom';
 import { Card } from 'antd';
 import { Button } from 'antd';
 
 const Landing = () => {
   return (
     <>
-      <Card title="Requires your consent" style={{ width: 300 }}>
-        <Button type="primary">Take me away</Button>
+      <Card
+        title="Requires your consent"
+        style={{ width: 300 }}
+        className="consent-card"
+      >
+        <a href="https://www.humanrightsfirst.org/">
+          <Button type="primary">Take me away</Button>
+        </a>
```

Comment:
Nice work redirecting to the humanrightsfirst homepage if they don't want to see potentially graphic material.